Remotion/ObjectBinding/Web.IntegrationTests/BocEnumValue/ReadOnlyBocEnumValueTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests;
+using Remotion.Web.Development.WebTesting.ControlObjects;
+using Remotion.Web.Development.WebTesting.ControlObjects.Selectors;
+using Remotion.Web.Development.WebTesting.ControlSelection;
+using Remotion.Web.Development.WebTesting.ExecutionEngine.PageObjects;
+using Remotion.Web.Development.WebTesting.FluentControlSelection;
+
+namespace Remotion.ObjectBinding.Web.IntegrationTests.BocEnumValue
+{
+  [TestFixture]
+  public class ReadOnlyBocEnumValueTest : IntegrationTest
+  {
+    [Test]
+    public void Validate_OnReadOnlyControl_CreatesValidationErrors ()
+    {
+      var home = Start();
+      var webButtonControlObject = home.GetControl(new LocalIDControlSelectionCommand<WebButtonControlObject>(new WebButtonSelector(), "ValidateButton"));
+
+      webButtonControlObject.Click();
+
+      var readOnly = home.EnumValues().GetByLocalID("MarriageStatusField_DropDownListReadOnly");
+      var validationErrors = readOnly.GetValidationErrorsForReadOnly();
+
+      Assert.That(validationErrors, Is.EqualTo(new [] {"Localized terrible marriage status."}));
+
+      readOnly = home.EnumValues().GetByLocalID("MarriageStatusField_DropDownListReadOnlyWithoutSelectedValue");
+      validationErrors = readOnly.GetValidationErrorsForReadOnly();
+
+      Assert.That(validationErrors, Is.EqualTo(new string [0]));
```

Comment:
Code style: asserting against `Is.Empty` is more idomatic precise/easier to read. I don't think we ever test against an explict empty array.

Reply:
Hm, somehow I had the thought in my head that we would prefer it that way. Maybe this was something more towards the Array.Empty<> thing